README.md

```diff
@@ -1,6 +1,394 @@
 # CNS Object Broker
-## For dynmically provisioning object storage compliant with an S3 interface.
 
 ## WARNING!
+The work in this project is a proof of concept and not intended for production.
 
-The work in this project is a proof of concept and not intended for anything more than that.
+## Utilize Kubernetes Service-Catalog to dynamically provision CNS Object Storage.
+
+## Overview
+A core feature of the Kubernetes system is the ability to provision a diverse
+offering of block and file storage on demand.
+This project seeks to demonstrate
+that by using the Kubernetes-Incubator's [Service-Catalog](https://github.com/kubernetes-incubator/service-catalog),
```

Comment:
s/"... service-catalog,"/"... service-catalog and a new object storage broker,"

Reply:
The CNS broker is just one of many that can provision S3 storage, so a new CNS object broker isn't required in that sense.  Will change it to
>This project seeks to demonstrate that by using the Kubernetes-Incubator's [Service-Catalog](https://github.com/kubernetes-incubator/service-catalog) and the CNS Object Broker, it is now possible to provision GlusterFS backed object storage on demand.

I think that brings it back to Gluster-Kubernetes.